Exoft.Gamification.Api.Common/Models/ErrorResponseModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Exoft.Gamification.Api.Common.Models
+{
+    public class ErrorResponseModel
+    {
+        public ErrorResponseModel(string message)
+        {
+            Message = message;
+        }
+
+        public ErrorResponseModel(Exception exception)
+        {
+            StringBuilder sb = new StringBuilder(exception.Message);
```

Comment:
Just do exception.ToString().... it ahs everything you need including stacktrace